\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=1.5in,footskip=0.75in]{geometry}

% use Unicode characters - try changing the option if you run into troubles with special characters (e.g. umlauts)
\usepackage[utf8]{inputenc}

% clean citations
\usepackage{cite}

% hyperref makes references clicky. use \url{www.example.com} or \href{www.example.com}{description} to add a clicky url
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% improves typesetting in LaTeX
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% text layout - change as needed
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.5in
\textheight 8.75in

% use adjustwidth environment to exceed text width (see examples in text)
\usepackage{changepage}

% adjust caption style
% \usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,singlelinecheck=off]{caption}

% remove brackets from references
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% headrule, footrule and page numbers
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
%\pagestyle{fancy}
%\fancyhf{}
%\rfoot{\thepage/\pageref{LastPage}}
%\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
%\fancyheadoffset[L]{2.25in}
%\fancyfootoffset[L]{2.25in}

% use \textcolor{color}{text} for colored text (e.g. highlight to-do areas)
\usepackage{color}

% define custom colors (this one is for figure captions)
\definecolor{Gray}{gray}{.25}

% this is required to include graphics
\usepackage{graphicx}

% use if you want to put caption to the side of the figure - see example in text
\usepackage{sidecap}

% use for have text wrap around figures
\usepackage{wrapfig}
\usepackage[pscoord]{eso-pic}
\usepackage[fulladjust]{marginnote}
\reversemarginpar

\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{verbatimbox}
\usepackage{relsize}
\usepackage{etoolbox}
\usepackage{natbib}


\bibliographystyle{natbibedit}


% document begins here
\begin{document}
\vspace*{0.35in}

% title goes here:
\begin{flushleft}
{\Large
\textbf\newline{Vaxrank: A computational tool for designing personalized cancer vaccines}
}
\newline
% authors go here:
\\
Alexander Rubinsteyn$^{\dagger,1}$,
Isaac Hodes$^1$,
Julia Kodysh$^1$,
Jeffrey Hammerbacher$^{1,2}$
\\
\bigskip

	$^1$ Department of Genetics and Genomic Sciences at Icahn School of Medicine at Mount Sinai \\
    $^2$ Department of Microbiology and Immunology, Medical University of South Carolina \\
	$^\dagger$ Contact: \href{alex@hammerlab.org}{alex@hammerlab.org} \\


\end{flushleft}

\section*{Abstract}
	Therapeutic vaccines targeting mutant tumor antigens (``neoantigens'') are an increasingly popular form of personalized cancer immunotherapy. Vaxrank is a computational tool for selecting neoantigen vaccine peptides from tumor mutations, tumor RNA data, and patient HLA type.  Vaxrank is freely available at \href{https://www.github.com/hammerlab/vaxrank}{www.github.com/hammerlab/vaxrank} under the Apache  2.0 open source license and can also be installed from the Python Package Index.


% the * after section prevents numbering

\section{Introduction}


Mutated cancer proteins recognized by T-cells have become known as ``neoantigens'' and are considered an essential component of a tumor-specific immune response ~\citep{neoantigens-finnigan, neoantigens-schumacher, neoantigens-gubin}. Therapeutic vaccination against neoantigens is an emerging experimental cancer therapy that attempts to mobilize an antigen-specific immune response against mutated tumor proteins~\citep{neovax-sharma, neovax-sahin}. Since few tumor mutations are shared between patients, neoantigen vaccines must be personalized therapies. A common approach for achieving personalization is high-throughput sequencing of tumor and normal patient samples followed by in-silico prioritization of mutated peptides that are likely to be presented on the surface of tumor cells by MHC (major histocompatibility complex) molecules.

Vaxrank is a tool for selecting mutated peptides for personalized therapeutic cancer vaccination. Vaxrank determines which peptides should be used in a vaccine from tumor-specific somatic mutations, tumor RNA sequencing data, and a patient's HLA type. These peptides can then be synthesized and combined with an adjuvant to attempt to elicit an anti-tumor T-cell response in a patient.

The sequence of each mutated protein is determined by assembling variant RNA reads. Mutant protein sequences are ranked using a scoring system which seeks to satisfy two objectives: choosing mutations that are abundant in the tumor and choosing those whose translated amino acid sequences contain likely MHC ligands. Additionally, Vaxrank considers surrounding non-mutated residues in a peptide to prioritize vaccine peptide candidates and to improve the odds of successful synthesis.

Vaxrank was designed for and is currently being used in the Personalized Genomic Vaccine Phase I clinical trial at the Icahn School of Medicine at Mount Sinai (NCT02721043)~\citep{pgv-pipeline}.


\begin{figure*}
	%\hspace*{-3cm}
	\includegraphics[scale=0.65, clip, trim=0 110 180 50]{vaxrank_diagram}
	\caption{Users provide tumor mutations, tumor RNA sequence data, and patient HLA type. These are used to determine mutant protein sequences and rank them according to expression and predicted MHC affinity.}
\end{figure*}

% \vspace{-5ex}
\section{Running Vaxrank}
To generate a Vaxrank vaccine report, the user must provide one or more files containing somatic variants (in VCF, MAF, or JSON format), aligned tumor RNA-seq reads (as an indexed BAM), and the HLA alleles to be used for MHC binding prediction:

\vspace{1ex}
\begin{verbatim}
    vaxrank
      --vcf somatic-variants.vcf
      --bam tumor-rna.bam
      --mhc-predictor netmhc
      --mhc-alleles H2-Kb,H2-Db
      --mhc-peptide-lengths 8-10
      --vaccine-peptide-length 21
      --min-alt-rna-reads 3
      --output-pdf-report vaccine-peptides.pdf
\end{verbatim}
\vspace{1ex}

The \verb|--mhc-predictor| argument controls which program is used to predict the affinity between a peptide-MHC pair. Vaxrank supports the use of locally installed instances of NetMHC~\citep{netmhc2016}, NetMHCpan~\citep{netmhcpan2007}, NetMHCcons~\citep{netmhccons}, MHCflurry~\citep{mhcflurry}, or a variety of web-based predictors through IEDB~\citep{iedb}. The \verb|--min-alt-rna-reads| argument controls the minimum number of RNA reads supporting a variant required to include that variant in the output report. In addition to quantifying tumor expression of a mutations, the RNA reads are used to phase adjacent variants when reconstructing the mutated coding sequence. A more complete list of options for input data, filtering, and output formats can be seen by running \verb|vaxrank --help|. Vaxrank's output can be formatted as PDF, plain-text, HTML, or an Excel spreadsheet. The output lists variants in ranked order along with vaccine peptide(s) containing that variant, predicted MHC ligands, number of supporting RNA reads, and sequence properties that affect manufacturability.

% \vspace{-3ex}
\section{Ranking Mutations}
A patient's coding mutations are ranked according to a score that combines each mutation's degree of expression and aggregate affinity of overlapping mutant peptides for that patient's MHC alleles.

%\vspace{-2ex}
\begin{align*}
\textit{RankingScore} 		&= \textit{ExpressionScore} \cdot \textit{TotalBindingScore} \\
\textit{ExpressionScore} 	&= \sqrt{\textit{\# RNA reads supporting variant allele}} \\
\textit{TotalBindingScore} 		&= \sum_{s \in \textit{subsequences}}\sum_{a \in \textit{alleles}}\textit{BindingScore}(s, a)
\end{align*}

The {\it BindingScore} function is, by default, a logistic transformation of the peptide-MHC binding affinity that loosely approximates the probability of T-cell response~\citep{Sette1994}. Alternatively, binding predictions can be scored using an affinity threshold (commonly $\leq 500$nM) or a threshold on the percentile rank of the affinity. Only subsequences which overlap mutant residues and do not occur in the reference proteome are considered as part of the {\it TotalBindingScore}.

%\vspace{-3ex}
\section{Manufacturability}
Vaxrank was designed under the assumption that its output will be used to make long peptides, due to their favorable immunological properties ~\citep{synthetic-long-peptides}. Unfortunately, long peptides are also more difficult to synthesize using traditional solid phase chemistry~\citep{PeptideChemistry}. To avoid known difficulties in synthesis, Vaxrank selects a window of amino acids around each mutation that minimizes the following undesirable properties:

\begin{quote}
\begin{enumerate}
	\item total number of cysteine residues
	\item $max(0, $ mean hydrophobicity of 7 residues at C-terminus$)$
	\item $max(0, $ mean hydrophobicity of any 7 amino acid window$)$
	\item glutamine, glutamic acid, or cysteine at N-terminus
	\item cysteine at C-terminus
	\item proline at C-terminus
	\item asparagine at N-terminus
	\item total number of asparagine-proline bonds
\end{enumerate}
\end{quote}
Manufacturability optimization does not affect the ranking of mutations but is only used for selecting which surrounding residues should be included.  In cases where a mutation  spans a ``difficult'' sequence (e.g. long hydrophobic stretch), minimizing these criteria  may fail to salvage manufacturability.

{\it Funding}: This work has been supported by the Icahn Institute and the Parker Institute for Cancer Immunotherapy.

 %\vspace{-3ex}

\bibliography{bibliography}

\end{document}
